rpcd/playbooks/roles/elasticsearch/tasks/elasticsearch_post_install.yml

```diff
@@ -48,21 +48,6 @@
   tags:
     - elasticsearch-post-install
 
-- name: Deploy ElasticSearch schema mapping script
-  template:
-    src: "mapping.sh.j2"
-    dest: "/opt/mapping.sh"
-    owner: "root"
-    group: "root"
-    mode: "0755"
-  tags:
-    - elasticsearch-post-install
-
-- name: Run ElasticSearch schema mapping script
-  shell: /opt/mapping.sh
-  tags:
-    - elasticsearch-post-install
-
```

Comment:
This mapping file actually causes elasticsearch to not start, and it seems that the new version of elasticsearch does a really good job of dynamic mapping.  We may need to add some custom mappings once the filter work begins but for now it is not needed.